internal/api/scim/integration_test/users_create_test.go

```diff
@@ -248,3 +235,9 @@ func TestCreateUser_scopedExternalID(t *testing.T) {
 	_, err = Instance.Client.UserV2.DeleteUser(CTX, &user.DeleteUserRequest{UserId: createdUser.ID})
 	require.NoError(t, err)
 }
+
+func TestCreateUser_anotherOrg(t *testing.T) {
+	org := Instance.CreateOrganization(Instance.WithAuthorization(CTX, integration.UserTypeIAMOwner), "fooBar", "foo@example.com")
+	_, err := Instance.Client.SCIM.Users.Create(CTX, org.OrganizationId, fullUserJson)
+	scim.RequireScimError(t, http.StatusNotFound, err)
```

Comment:
Why does a create use in another org return a 404?

Reply:
As far as I understand it an OrgAdmin should not be allowed to create a user in another org without a membership/grant. This returns 404 since the membership is not found. I'm not sure whether this should be a 403 instead, but this seems to be out of scope for this PR (adjustments would probably be needed in `authz.CheckUserAuthorization` to wrap the not found error in a auth related error).

Reply:
Yeah true, that's outside of scope, totally missed that.